### PR TITLE
Add missing text reference css class

### PR DESCRIPTION
**Pull request by tchow8**
# Description

[DIV-4225](https://tools.hmcts.net/jira/browse/DIV-4225)

Fixes missing css styling from reform-pattern-library.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "moment": "^2.22.1",
     "os": "^0.1.1",
     "paths": "^0.1.1",
+    "reform-pattern-library": "https://github.com/hmcts/reform-pattern-library.git",
```
Comment (hosainnet) [inline package.json]: We removed this because it's deprecated. Do we know what styles? Is it easy to migrate those to `look-and-feel` for example instead?

Comment (tchow8) [reply package.json]: Took the style from reform-pattern-library.

**Comment by tchow8** [comment]
Applying this change to look-and-feel. I'll modify this Pr to use an updated look-and-feel version after that gets merged in.

**Comment by nsingamsetty** [comment]
This is tested and working as expected. Ready to merge. As this is a bug no Toggle is required. Refer ticket DIV-4225 for more details.

**Pull request merged**